pkg/limits/ingest_limits.go

```diff
@@ -82,6 +98,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	cfg.LifecyclerConfig.RegisterFlagsWithPrefix("ingest-limits.", f, util_log.Logger)
 	f.BoolVar(&cfg.Enabled, "ingest-limits.enabled", false, "Enable the ingest limits service.")
 	f.DurationVar(&cfg.WindowSize, "ingest-limits.window-size", 1*time.Hour, "The time window for which stream metadata is considered active.")
+	f.DurationVar(&cfg.RateWindow, "ingest-limits.rate-window", 5*time.Minute, "The time window for rate calculation. This should match the window used in Prometheus rate() queries for consistency.")
```

Comment:
Shouldn't it be the other way around? You should adjust your Prometheus queries to match the rate window we decide for the limits service?

Reply:
It is a matter of interpretation I think. We mean the same here. It is a note for the user to adapt the PromQL queries actually. Can you suggest better wording?